README.md

```diff
@@ -100,6 +100,51 @@ Deferring the display of markup is typically done in the following usage pattern
 
 - The first time a user visits a page that contains a time-consuming piece of markup to generate, a loading indicator is displayed. When the markup is finished building on the server, it's stored in memcache and sent to the browser to replace the include-fragment loader. Subsequent visits to the page render the cached markup directly, without going through a include-fragment element.
 
+### CSP Trusted Types
+
+You can call `setCSPTrustedTypesPolicy(policy: TrustedTypePolicy | Promise<TrustedTypePolicy> | null)` from JavaScript to set a [CSP trusted types policy](https://web.dev/trusted-types/), which can perform (synchronous) filtering or rejection of the `fetch` response before it is inserted into the page:
+
+```ts
+import { setCSPTrustedTypesPolicy } from "include-fragment-element";
+import DOMPurify from "dompurify"; // Using https://github.com/cure53/DOMPurify
+
+// This policy removes all HTML markup except links.
+const policy = trustedTypes.createPolicy("links-only", {
+  createHTML: (htmlText: string) => {
+    return DOMPurify.sanitize(htmlText, {
+      ALLOWED_TAGS: ["a"],
+      ALLOWED_ATTR: ["href"],
+      RETURN_TRUSTED_TYPE: true,
+    });
+  },
+});
+setCSPTrustedTypesPolicy(policy);
+```
+
+The policy has access to the `fetch` response object. Due to platform constraints, only synchronous information from the response (in addition to the HTML text body) can be used in the policy:
+
+```ts
+import { setCSPTrustedTypesPolicy } from "include-fragment-element";
```

Comment:
I believe this documentation needs updating now.

Reply:
Nice catch! Addressed in https://github.com/github/include-fragment-element/pull/81/commits/4aa007545d3b18ff81f62d41b900fea67050399f